queue_services/entity-filer/src/entity_filer/filing_processors/incorporation_filing.py

```diff
@@ -74,11 +75,16 @@ def process(business: Business, filing: Dict, app: Flask = None):  # pylint: dis
                         party = create_party(party_info=party_info)
                         for role_type in party_info.get('roles'):
                             role = {
-                                'roleType': role_type,
+                                'roleType': role_type.get('roleType'),
```

Comment:
@kialj876 Could you please confirm whether this is correct. As per the sample data from the business schema, this is needed